WHERE clause combined with procedure call is allowed on the system database (#510)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors/42N17.adoc

You try to run a query that contains an allowed procedure but an unallowed clause.
For example, the `YIELD` clause in the following query is allowed for procedure calls, but the `WHERE` clause is not:

[NOTE]
====
This example is valid only for versions prior to Neo4j 2026.06.

> From 2026.06 onwards, the `WHERE` clause, when combined with procedure calls, is allowed on the `system` database, and the following example query works as expected.

Reviewer: > From 2026.06 onwards, the `WHERE` clause, **when combined with procedure calls**, is allowed on the `system` database, and the following example query works as expected.

Not sure about this wording. I can't add a `WITH * WHERE ...` even if I have a procedure call, it needs to be the `WHERE` that is part of the procedure call (or it's yield) to be allowed.

Maybe:
```suggestion
From 2026.06 onwards, the `WHERE` clause, when part of procedure calls, is allowed on the `system` database, and the following example query works as expected.
```

To solve this, run the query against a standard database.

[NOTE]
====

> In contrast to procedure calls, the `WHERE` clause is allowed for the `SHOW` and `TERMINATE` commands.

Reviewer: this should either also mention the release, or just be removed. Since from 2026.06 the procedure calls also allow WHERE

Author: Yeah, makes sense. I was wondering what to do with it.
